react/javascript/tile-visualization/src/components/DashboardTile/DashboardTile.js

```diff
@@ -0,0 +1,124 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2022 Looker Data Sciences, Inc.
```

Comment:
```suggestion
 Copyright (c) 2023 Looker Data Sciences, Inc.
```

and so on, if you want to take the update